docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -10,7 +10,7 @@ For the Impatient
 These instructions will create a developer install of Pulp on a dedicated
 development instance.
 
-* Start a Fedora 20 x86_64 instance that will be dedicated for development with
+* Start a RHEL 7 or Fedora 20 x86_64 instance that will be dedicated for development with
```

Comment:
We need to create Fedora-21 before:
https://repos.fedorapeople.org/repos/pulp/pulp/stable/2/

Otherwise the installation fails, yum.repos.d contais:
https://repos.fedorapeople.org/repos/pulp/pulp/stable/latest/fedora-$releasever/$basearch/